src/main/java/duke/DialogBox.java

```diff
@@ -0,0 +1,45 @@
+package duke;
```

Comment:
minor nit but might want to add new line here for consistency across the files

Reply:
Nice catch, I completely did not notice this.